src/dinit.cc

```diff
@@ -491,6 +610,21 @@ int dinit_main(int argc, char **argv)
         }
     }
 
+    // Set the log format correctly to buffer any log messages
+    early_init_log(log_is_syslog);
+
+#if SUPPORT_SELINUX
+    // Error exit if we are PID 1 and fail to load the selinux policy.
```

Comment:
If we are system manager, isn't it? Regardless of PID 1 status?

Reply:
Indeed, sorted :)